test/integration/node/lifecycle_test.go

```diff
@@ -50,12 +51,39 @@ func TestEvictionForNoExecuteTaintAddedByUser(t *testing.T) {
 
 	tests := map[string]struct {
 		enablePodDisruptionConditions bool
+		enableSeparateTaintManager    bool
+		startStandaloneTaintManager   bool
+		wantPodEvicted                bool
 	}{
-		"Test eviciton for NoExecute taint added by user; pod condition added when PodDisruptionConditions enabled": {
+		"Test eviction for NoExecute taint added by user; pod condition added when PodDisruptionConditions enabled": {
 			enablePodDisruptionConditions: true,
+			enableSeparateTaintManager:    false,
+			startStandaloneTaintManager:   false,
+			wantPodEvicted:                true,
 		},
-		"Test eviciton for NoExecute taint added by user; no pod condition added when PodDisruptionConditions disabled": {
+		"Test eviction for NoExecute taint added by user; no pod condition added when PodDisruptionConditions disabled": {
 			enablePodDisruptionConditions: false,
+			enableSeparateTaintManager:    false,
+			startStandaloneTaintManager:   false,
+			wantPodEvicted:                true,
+		},
+		"Test eviction for NoExecute taint added by user; separate taint manager disabled": {
+			enablePodDisruptionConditions: false,
+			enableSeparateTaintManager:    false,
+			startStandaloneTaintManager:   false,
+			wantPodEvicted:                true,
+		},
```

Comment:
This is duplicated with L64~L69.

Reply:
Nice catch! Should have been de-duped now in 4ae3ba39e4c